ui.frontend/src/utils.js

```diff
@@ -219,8 +219,12 @@ class Utils {
      */
     static updateId(htmlElement, oldId, newId) {
         let elementWithId = htmlElement.querySelectorAll("#" + oldId)[0];
+        let elementWithErrorId = htmlElement.querySelector("#" + `${oldId}__errormessage`);
```

Comment:
Rename the variable to "errorElementWithId"

Reply:
Renamed!